templates/e-commerce/product-details.amp.html

```diff
@@ -22,6 +22,13 @@
   {{#header}}{{> ../../components/navbar/navbar.snip.html}} {{/header}} {{#main}}
   <main id="content" role="main" class="main">
     {{#product-details}}
+    <amp-state id="cart">
```

Comment:
Separate amp-state for cart. It's not related to the current product on the page, thats why it wasn't added to the product state.